symbulate/random_variables.py

```diff
@@ -46,11 +71,18 @@ def check_same_probSpace(self, other):
             self.probSpace.check_same(other.probSpace)
 
     def apply(self, function):
-        """
+        """Transform a random variable by a function.
+
         Args:
-            function: function to apply to the random variable (e.g., log, sqrt, exp)
+          function: function to apply to the random variable
         
-        Input function is applied to the output results.
+        Examples:
```

Comment:
Maybe it's better to start without lambda?

X = RV(Exponential(1))
Y = X.apply(log)

Note: For most standard functions, you can apply the function to
 the random variable directly. For example, in the example above,
 Y = log(X) would have been equivalent and more readable.

# User defined functions can also be applied
def g(x):
  return log(x ** 2)

X = RV(Exponential(1))
Y = X.apply(g)

# In this example Y could also be defined via either of the following
Y = X.apply(lambda x: log(x ** 2)) 
Y = log(X ** 2)